src/include/bootloader/routines/dialogs.rb

```diff
@@ -128,7 +128,7 @@ def MainDialog
 
       )
       if ret != :back && ret != :abort && ret != :cancel
-        @return_tab = CWMTab.LastTab
+        @return_tab = CWMTab.LastTab || "tab"
```

Comment:
So "tab" is both the id of an existing Tab widget, and a fake name of a non-existing tab?